style.css

```diff
@@ -433,3 +446,133 @@ hr.solid {
   margin-top: 10px;
   margin-bottom: 10px;
 }
+
+.contact-me {
+  width: 375px;
+  height: 820px;
+  background: #6070ff;
+  background-image: url(./images/contact\ form\ .png);
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: 100% 100%;
+  border-top-left-radius: 30px;
+  padding-left: 24px;
+  padding-right: 24px;
+  padding-top: 103px;
+  gap: 12px;
+  position: relative;
+}
+
+.contact-me h3 {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  color: #fff;
+}
+
+.contact-me p {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #ebebff;
+}
+
+form {
+  padding-top: 40px;
```

Comment:
As requested by the previous reviewer, kindly remove the large margin of white space around your page. I see that you made an effort to remove it from the right side but it is still there, has even extended to the left side as shown below.  This is as a result of the fixed width you have placed in almost all of the sections. This prevents the containers of the sections from expanding and becoming responsive, hence they maintain the assigned fixed widths even when you increase the screen widths. 

![Screenshot from 2023-01-26 11-07-19](https://user-images.githubusercontent.com/65233860/214820985-94c6e852-36dc-418d-a9f1-71ad3c7bb1d3.png)

Reply:
please I need more explanation on this because I am doing this task for the mobile version I have used all height and width from figma and the previous reviewer told me to put the portfolio in the center I have even done it so the problem I have is how I can remove those spaces as  width and height are based on figma requirements so I don't know if I have to do it with the screen of width 100% while I am doing task for mobile please

Reply:
thanks for your feedback  @jerryowusu check my comment I need more clarification